.sisyphus/dependency-maps/2026-05-13-sse-interleaved-response.md

```
@@ -0,0 +1,35 @@
+| 2 | blocks.py EventItem description 추가 | quick | [] | [1, 8] |
+| 3 | place_search_node description + 순서 변경 | deep | [1, 2] | [5] |
+| 4 | place_recommend_node 동일 패턴 | deep | [3] | [6] |
+| 5 | event_search_node description + 순서 변경 | deep | [1, 2] | [3] |
+| 6 | event_recommend_node 동일 패턴 | deep | [5] | [4] |
+| 7 | response_builder_node 순서 업데이트 | quick | [3, 4, 5, 6] | [] |
+| 8 | general_node 마크다운 개선 | quick | [] | [1, 2] |
+| 9 | 통합 검증 | quick | [7, 8] | [] |
+
+## 그룹 + 실행 순서
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)